swagger/api.yaml

```diff
@@ -58,6 +58,20 @@ paths:
   "/enterprise/v1/enterprise-customer/{uuid}/learner-summary":
     $ref: "https://raw.githubusercontent.com/edx/edx-enterprise-data/8fc95e4/api.yaml#/endpoints/v1/enterpriseCustomerLearnerSummary"
 
+  #Registrar
+  "/registrar/v0/jobs/{job_id}":
+    $ref: "https://raw.githubusercontent.com/edx/registrar/21f6816e2e0d40cd1d65a34ea21723f1bfa3f1b1/.api-generated.yaml#/paths/~1v0~1jobs~1{job_id}~1"
```

Comment:
Can we find out if the really needs to be a specific commit?  I'd recommend pointing to master so we can pick up changes automatically unless there is a good reason this needs to be a manual process.